  <launch>
    <!-- Decompose Node-->
    <node pkg="homog_track" type="decomp_node" name="decomp_node" />
  </launch>
